time/src/main/java/net/jqwik/time/internal/properties/arbitraries/DefaultPeriodArbitrary.java

```diff
@@ -0,0 +1,75 @@
+package net.jqwik.time.internal.properties.arbitraries;
+
+import java.time.*;
+
+import org.apiguardian.api.*;
+
+import net.jqwik.api.*;
+import net.jqwik.api.arbitraries.*;
+import net.jqwik.time.api.arbitraries.*;
+
+import static org.apiguardian.api.API.Status.*;
+
+@API(status = INTERNAL)
+public class DefaultPeriodArbitrary extends ArbitraryDecorator<Period> implements PeriodArbitrary {
+
+	private int yearsMin = Integer.MIN_VALUE;
+	private int yearsMax = Integer.MAX_VALUE;
+
+	private int monthsMin = Integer.MIN_VALUE;
+	private int monthsMax = Integer.MAX_VALUE;
+
+	private int daysMin = Integer.MIN_VALUE;
+	private int daysMax = Integer.MAX_VALUE;
+
+	@Override
+	protected Arbitrary<Period> arbitrary() {
+
+		IntegerArbitrary years = Arbitraries.integers().between(yearsMin, yearsMax);
+		IntegerArbitrary months = Arbitraries.integers().between(monthsMin, monthsMax);
+		IntegerArbitrary days = Arbitraries.integers().between(daysMin, daysMax);
+
+		return Combinators.combine(years, months, days).as((y, m, d) -> Period.ofYears(y).plusMonths(m).plusDays(d));
+
+	}
```

Comment:
@jlink Can you please provide feedback to this type of generation of periods?

- Maybe days should be between 0 and 30 by default?
- Maybe months should be between 0 and 11 by default?

Reply:
I think those defaults are reasonable